<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rent_form</class>
 <widget class="QWidget" name="rent_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>486</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>864</width>
    <height>486</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>864</width>
    <height>486</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>821</width>
     <height>441</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
    <property name="spacing">
     <number>15</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,22">
      <property name="spacing">
       <number>20</number>
      </property>
      <item>
       <widget class="QLabel" name="rent_label">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Segoe UI&quot;;
</string>
        </property>
        <property name="text">
         <string>Форма нового договора аренды</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QFormLayout" name="formLayout">
        <property name="verticalSpacing">
         <number>20</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="new_cust_l">
          <property name="text">
           <string>Новый клиент?</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QRadioButton" name="new_cust_v">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="name_l">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="name_v">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="lastname_l">
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="lastname_v">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="surname_l">
          <property name="text">
           <string>Отчество</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="surname_v">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="pass_l">
          <property name="text">
           <string>Номер паспорта</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="pass_v">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="start_date_l">
          <property name="text">
           <string>Начало аренды</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QDateEdit" name="start_date_v"/>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="end_date_l">
          <property name="text">
           <string>Окончание аренды</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QDateEdit" name="end_date_v"/>
        </item>
        <item row="7" column="1">
         <widget class="QPushButton" name="add_rent_btn">
          <property name="text">
           <string>Готово</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="msg_box">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10">
      <item>
       <widget class="QLabel" name="cust_table_lable">
        <property name="styleSheet">
         <string notr="true">font: 14pt &quot;Segoe UI&quot;;

</string>
        </property>
        <property name="text">
         <string>База данных клиентов</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="cust_table">
        <attribute name="horizontalHeaderVisible">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>60</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>ID</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>ФИО</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
